source/2015-07-14-3-planning-pokers-in-3-days.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+title: 3 planning pokers in 3 days
+date: 2015-07-14 10:41 UTC
+tags:
+---
+
+During our Kalamata Klassenlager there were quite some pitches to try out
+new technologies. As the week progressed three groups around three different
+main technologies seemed to emerge: React / Hoodie, Volt and Meteor.
+Instead of implementing a todo app for the millionth time each group
+implemented a planning poker in their respective technology.
+An online planning poker could actually be something useful.
+
+## React / Hoodie
```

Comment:
Could someone from the React / Hoodie Team (@huerlisi, ...) write some sentences here?